bindings/python/test/trajectory/test_segment.py

```diff
@@ -135,6 +135,64 @@ def thruster_dynamics() -> Thruster:
     )
 
 
+@pytest.fixture
+def segment_solution(dynamics: list[Dynamics], state: State) -> Segment.Solution:
+    return Segment.Solution(
+        name="A Segment",
+        dynamics=dynamics,
+        states=[
+            state,
+        ],
+        condition_is_satisfied=True,
+        segment_type=Segment.Type.Coast,
+    )
+
+
+class TestSegmentSolution:
+    def test_constructors(
+        self,
+        segment_solution: Segment.Solution,
+    ):
+        assert segment_solution is not None
+        assert segment_solution.name is not None
+        assert segment_solution.dynamics is not None
+        assert segment_solution.states is not None
+        assert segment_solution.condition_is_satisfied is not None
+        assert segment_solution.segment_type is not None
+
+    def test_accessors(
+        self,
+        segment_solution: Segment.Solution,
+    ):
+        assert segment_solution.access_start_instant() is not None
+        assert segment_solution.access_end_instant() is not None
+        assert segment_solution.get_interval() is not None
+        assert segment_solution.get_initial_mass() is not None
+        assert segment_solution.get_final_mass() is not None
+        assert segment_solution.get_propagation_duration() is not None
+
+    def test_compute_delta_v(
+        self,
+        segment_solution: Segment.Solution,
+    ):
+        assert segment_solution.compute_delta_v(1500.0) is not None
+
+    def test_compute_delta_mass(
+        self,
+        segment_solution: Segment.Solution,
+    ):
+        assert segment_solution.compute_delta_mass() is not None
+
+    # def test_extract_maneuvers(
+    #     self,
+    #     dynamics: list,
+    #     state: State,
+    # ):
+    #     segment_solution: Segment.Solution = Segment.Solution(
+    #         name="A Segment",
+    #         dynamics=dynamics,
```

Comment:
What's this doing here? Will you implement it?

Reply:
Done!